spec/services/ccms/requestors/case_add_requestor_xml_blocks/non_passported_attributes_spec.rb

```diff
@@ -462,6 +462,13 @@ module Requestors # rubocop:disable Metrics/ModuleLength
           end
         end
 
+        context 'GB_INPUT_C_13WP3_4A' do
+          it 'is omitted' do
+            block = XmlExtractor.call(xml, :global_means, 'GB_INPUT_C_13WP3_4A')
+            expect(block).not_to be_present, "Expected block for attribute GB_INPUT_C_13WP3_4A not to be generated, but was \n #{block}"
```

Comment:
Is the error text supposed to be there? I've not seen that pattern on other tests.

Reply:
removed it as it is used for debugging (printing out the block)